help/destinations/catalog/social/tiktok.md

```diff
@@ -0,0 +1,114 @@
+---
+title: TikTok connection // Replace with the name of destination
+description: Create segments from your customer lists from people who've visited your site or people who have already interacted with your content on TikTok and quickly push the desired segment to TikTok using TikTok's real-time streaming API to use for ad targeting. // Replace with the paragraph in the Overview section
+---
+
+# TikTok connection
+
+## Overview {#overview}
+
+Create segments from your customer lists from people who've visited your site or people who have already interacted with your content on TikTok and quickly push the desired segment to TikTok using TikTok's real-time streaming API to use for ad targeting. Go to [TikTok's website](https://ads.tiktok.com/help/article/audiences?lang=en) for more info. 
```

Comment:
Minor, just a suggestion to reword the last sentence as it might come across as a bit too informal:

Visit [TikTok's business help center](https://ads.tiktok.com/help/article/audiences?lang=en) for more information.